dvc/repo/fetch.py

```diff
@@ -76,24 +76,24 @@ def _fetch_external(self, repo_url, repo_rev, files):
     cache_dir = self.cache.local.cache_dir
     try:
         with external_repo(repo_url, repo_rev, cache_dir=cache_dir) as repo:
-            cache = NamedCache()
-            for name in files:
-                try:
-                    out = repo.find_out_by_relpath(name)
-                except OutputNotFoundError:
-                    failed += 1
-                    logger.exception(
-                        "failed to fetch data for '{}'".format(name)
-                    )
-                    continue
-                else:
-                    cache.update(out.get_used_cache())
-
             with repo.state:
-                try:
-                    return repo.cloud.pull(cache), failed
-                except DownloadError as exc:
-                    failed += exc.amount
+                cache = NamedCache()
+                for name in files:
+                    try:
+                        out = repo.find_out_by_relpath(name)
+                    except OutputNotFoundError:
+                        failed += 1
+                        logger.exception(
+                            "failed to fetch data for '{}'".format(name)
+                        )
+                        continue
+                    else:
+                        cache.update(out.get_used_cache())
+
+                    try:
```

Comment:
Wrong indentation, it should be one level to the left.

Reply:
fixed! thanks.